www/docs/en/dev/config_ref/index.md

```diff
@@ -245,6 +244,9 @@ platform. See [Customize icons topic](images.html) for more information.
    PopupBlocker(string) | *Default: enable* <br/> ==BlackBerry== <br/> Enables the popup blocker, which prevents calls to window.open(). By default, popups display in a child browser window. Setting the preference to enable prevents it from displaying at all.
    SetFullscreen(boolean) | *Default: false* <br/> ==Android== <br/> Same as the Fullscreen parameter in the global configuration of this xml file. This Android-specific element is deprecated in favor of the global Fullscreen element, and will be removed in a future version.
    ShowTitle(boolean) | *Default: false* <br/> ==Android== <br/> Show the title at the top of the screen.
+   SplashScreen(string) | *Default: splash* <br/> ==Android== <br/> The name of the file minus its extension in the ```res/drawable``` directory. Various assets must share this common name in various subdirectories.
```

Comment:
We do not document SplashScreen config properties as they are considered part of the SplashScreen plugin - even though the code is part of each platform. We should probably mention one of them here without providing details (so as not to duplicate the docs) and say that splashscreen plugin is required for them to work and they are documented there!